src/Components/CreateImageWizard/formComponents/ActivationKeys.js

```diff
@@ -91,7 +91,7 @@ const ActivationKeys = ({ label, isRequired, ...props }) => {
       change('subscription-server-url', 'subscription.rhsm.stage.redhat.com');
       change('subscription-base-url', 'https://cdn.stage.redhat.com/');
     }
-  }, []);
+  }, [change, isProd]);
```

Comment:
Same kind of comment here for the `useEffect` dependencies:

`change` shouldn't endup in the list, only `isProd()` is making sense here.
The way we're using it is a sign of an antipattern however, I'm not sure with this fix, I would rather let it be in the form it was before.

Reply:
but without the 'change' I still get this warning message - /Users/michalgold/git/edge-management/image-builder-project/image-builder-frontend/src/Components/CreateImageWizard/formComponents/ActivationKeys.js
  94:6  warning  React Hook useEffect has a missing dependency: 'change'. Either include it or remove the dependency array  react-hooks/exhaustive-deps